Dockerfile

```diff
@@ -0,0 +1,79 @@
+# Sources: https://sourcery.ai/blog/python-docker/ and
+# and https://github.com/pypa/pipenv/blob/2bf70b74167868133809a926aa6393438fb06db4/docs/basics.rst#-pipenv-and-docker-containers
+FROM python:3.10-slim as base
+
+# Setup env
+ENV LANG C.UTF-8
+ENV LC_ALL C.UTF-8
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONFAULTHANDLER 1
+
+
+FROM base AS pipenv
+
+# Install pipenv and compilation dependencies
+RUN pip3 install --user --no-cache-dir pipenv
+RUN apt-get update && apt-get install -y --no-install-recommends gcc python-dev git
+
+# Tell pipenv to create venv in the current directory
+ENV PIPENV_VENV_IN_PROJECT=1
+
+
+# ----- Build Streamlit App -----
+FROM pipenv AS streamlit-app-python-deps
+
+# Install python dependencies in /.venv
+# COPY streamlit_app/Pipfile .
+# COPY streamlit_app/Pipfile.lock .
+# RUN /root/.local/bin/pipenv sync
```

Comment:
I want to keep this, because the plan it to eventually return to using `pipenv` to install dependencies in the container. The reason it isn't currently used is due to a bug in `pipenv`